components/shooter.py

```diff
@@ -0,0 +1,32 @@
+from magicbot import tunable
+import wpilib
+
+
+class Shooter:
+    shooter_motor: wpilib.Talon
+    shoot_speed = tunable(1.0)  # speed is tunable via NetworkTables
+
+    def __init__(self):
+        self.deployed = False
+
+    def shoot(self):
+        pass
+
+    def stop(self):
+        pass
+
+    def deploy(self):
+        """Causes the shooter motor to spin"""
+        self.deployed = True
+
+    def is_ready(self):
+        return True
+
```

Comment:
I think it's safe to say we need more feedback methods.
I'm thinking of two feedback methods:

1. actual_velocity
2. flywheel error